logging/exp/alias.go

```diff
@@ -0,0 +1,138 @@
+// Copyright 2023 The Authors (see AUTHORS file)
```

Comment:
@yolocs here's what I meant about the aliases. This should make it so users don't need to directly import the `slog` package (much like they don't need to import `zap` today). It's still a dependency, but the import is masked.

Reply:
Awesome!